domain/modelconfig/service/service.go

```diff
@@ -344,6 +346,7 @@ func (s *Service) updateModelConfigValidator(
 			}),
 			validators.SecretBackendChecker(&dummySecretsBackendProvider{}),
 			validators.AuthorizedKeysChange(),
+			validators.ContainerNetworkingMethodChange(),
```

Comment:
I think we should add a validator for the string the user is setting as well on both update and set.

Reply:
done